### ISSUE TITLE
Form group on the “My Language” screen is not programmatically associated with its group label [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#272]
The “Choose your language” form group label is not announced upon focusing a control in the form group. For example, the “English” checkbox is announced as “English, selected, checkbox”.

![](https://user-images.githubusercontent.com/4437084/92059781-1d279c80-ed60-11ea-8148-f1317612abd2.png)

Use the accessibilityLabel property to include the group label in the accessible name of each control. Remove the group label from the reading order using the accessibilityElementsHidden, as it is redundant when included in the accessibilityLabel of each checkbox.

```javascript
<View accessibilityElementsHidden={true}>
  <Text>Choose your language:</Text>
</View>

<TouchableOpacity
  onPress={…}
  accessibilityLabel='English, Choose your language?'
  accessibilityRole='checkbox'
  accessibilityState={{selected: true}}
/>
```

**Comment by floridemai** [comment]
@willslone This has now been addressed with the same mention. The label does not include "Choose your language?". Please re-assign and I'll update if that's the case.
